### PR TITLE
Additional cleanup

### PR DESCRIPTION
**Pull request by apasel422**
r? @Gankro

### CODE REVIEW

**Comment by apasel422** [comment]
Ping.

File: src/lib.rs [hunk 1]
```diff
@@ -272,6 +288,22 @@ impl<T> LinkedList<T> {
...
+    ///
+    /// After this method returns, `self` contains the elements that previously lay in the range
+    /// `[0, index)`, and the returned list contains the elements that previously lay in the range
+    /// `[index, len)`.
+    ///
+    /// # Panics
+    ///
+    /// Panics if the given index is greater than the length of the list.
+    pub fn split_off(&mut self, index: usize) -> Self {
+        assert!(index <= self.len());
```
Comment (Gankro) [inline src/lib.rs]: Can you give a message?

**Comment by Gankro** [comment]
r=me with message

**Comment by apasel422** [comment]
Updated with the message used by `std`.

**Pull request merged**